README.md

```diff
@@ -680,15 +688,15 @@ For this CEP-78 reference implementation in particular, the events schema is the
 | Migration       | -                                                                    |
 | VariablesSet    | -                                                                    |
 
-Token identifiers are stored under the `CLType` `Any` and the encoding depends on `NFTIdentifierMode`:.
+Token identifiers are stored under the `CLType` `Any` and the encoding depends on the `NFTIdentifierMode` modality:
 
-* `NFTIdentifierMode::Ordinal`: the `token id` is encoded as a byte `0x00` followed by a `u64` number.
-* `NFTIdentifierMode::Hash`: the `token_id` is encoded as a byte `0x01` followed by a `String`.
+- `NFTIdentifierMode::Ordinal`: the `token id` is encoded as a byte `0x00` followed by a `u64` number.
+- `NFTIdentifierMode::Hash`: the `token_id` is encoded as a byte `0x01` followed by a `String`.
```

Comment:
Two paragraphs above should be removed because the `token_id` type was changed to `String`, which can be parsed by a generic parser

Reply:
Thanks please check that commit, let me know if I missed something

https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/181/commits/ed86eab5d148af6c9cabbbcb48bd1dac01d21521